### DISCUSSION TITLE
Parallel suites execution

### DISCUSSION THREAD

**Discussion by m3rlinux** [elabit/robotmk#200] [Ideas]
Hi Simon!
Did you plan a release with "agent_parallel" as you mentioned in #59?
Thanks in advance.

**Comment by simonmeggle** [comment]
See my talk at the CMK conf #8: https://youtu.be/d7e1h27Mz6M?t=17711
It will be the first thing I will add to Robotmk soon and it will be available for Robotmk Core (Open Source) and Enterprise as well.

**Comment by m3rlinux** [comment]
Oh! Yeah!
I'm so excited in earing this.
I can not wait! :) 
Thanks!

**Comment by simonmeggle** [comment]
Will be done in #205

**Reply by m3rlinux** [reply]
Hi Simon,
has it been implemented?
Thanks!

**Reply by simonmeggle** [reply]
Hi, 
> has it been implemented?

not yet. But it is the very first big topic on the roadmap. 
Because of the fact that this requires major changes under the hood, I am currently refactoring the whole thing to make the code more maintainable and extendable. Be patient :-)

**Reply by m3rlinux** [reply]
Ok! I hope it will be my Christmas gift ;)

**Reply by simonmeggle** [reply]
could rather be in the easter nest 🧆
😌

**Reply by m3rlinux** [reply]
Doh! 🤦‍♂️
Ok, I'll try to survive until easter day.
But, do not put un egg on it 😜

**Comment by m3rlinux** [comment]
Hi Simon,
seems you moved to Jira (as you said in #205), is there a way to follow the work progresses?
Thanks!

**Reply by simonmeggle** [reply]
Hi, meanwhile this is implemented. We have now a Robotmk Scheduler which gets started as a "Agent Extension" (new feature), just like a damon. The Scheduler is able to tun suites in individual intervals. Headful execution is also implemented; no need to have an "external" mode anymore. 
Stay tuned, CMK 2.3 will bring it to you :-)
